share/spack/gitlab/cloud_pipelines/stacks/python/spack.yaml

```diff
@@ -0,0 +1,1829 @@
+spack:
+  view: false
+  concretization: separately
+
+  config:
+    concretizer: clingo
+    install_tree:
+      root: /home/software/spack
+      padded_length: 512
+      projections:
+        all: '{architecture}/{compiler.name}-{compiler.version}/{name}-{version}-{hash}'
+
+  specs:
+    - adios2
```

Comment:
`adios2` is already built as a root spec in the `e4s` and `data-vis-sdk` stacks, some of these other specs are built elsewhere too.  I was expecting this stack to include only `python` and `py-...` packages, but if you need to repeat these other builds for some reason, that's fine.  Just pointing out that they'll be brought in as dependencies if needed, and they're built explicitly elsewhere.

Reply:
Yep, I'm adding anything that extends Python (I actually needed to add `+python` on this one and others). This will be removed eventually so I'm not too worried about duplication.